vispy/app/backends/_qt.py

```diff
@@ -138,14 +138,27 @@ def _check_imports(lib):
 
 # Properly log Qt messages
 # Also, ignore spam about tablet input
-def message_handler(msg_type, msg):
+def message_handler(*args):
+
+    if qt_lib in ("pyqt4", "pyside"):
+        msg_type, msg = args
+    elif qt_lib == "pyqt5":
+        msg_type, context, msg = args
+    elif qt_lib:
+        raise RuntimeError("Invalid value for qt_lib %r." % qt_lib)
+    else:
+        raise RuntimeError("Module backends._qt ",
+                           "should not be imported directly.")
+
     if msg == ("QCocoaView handleTabletEvent: This tablet device is "
                "unknown (received no proximity event for it). Discarding "
                "event."):
         return
     else:
-        logger.warning(msg)
-
+        if qt_lib in ("pyqt4", "pyside"):
+            logger.warning(msg.decode())
+        elif qt_lib == "pyqt5":
+            logger.warning(msg)
```

Comment:
Could you please elaborate ? What should I put for `string_types` since `unicode` only exist in Python 2.